File: app/src/main/java/com/nimtego/itunes/view/toast/SimpleToastAlarm.java
package com.nimtego.itunes.view.toast;

import android.content.Context;
import android.widget.Toast;

public class SimpleToastAlarm implements ToastAlarm {

    private Context mContext;

    public SimpleToastAlarm(Context commonView) {
        mContext = commonView;
    }

    @Override
    public void destroy() {
        mContext = null;
    }

    @Override
    public void message(String message) {
        Toast toast = Toast.makeText(mContext.getApplicationContext(),
                message, Toast.LENGTH_SHORT);
        toast.show();
    }
}

File: app/src/main/java/com/nimtego/plectrum/presentation/ui/widget/toast/SimpleToastAlarm.kt
package com.nimtego.plectrum.presentation.ui.widget.toast

import android.content.Context
import android.widget.Toast

class SimpleToastAlarm(private var mContext: Context?) : ToastAlarm {

    override fun destroy() {
        mContext = null
    }

    override fun message(message: String) {
        val toast = Toast.makeText(mContext!!.applicationContext,
                message, Toast.LENGTH_SHORT)
        toast.show()
    }
}
